package.json

```diff
@@ -81,24 +79,24 @@
     "random-word-slugs": "^0.1.6",
     "redis": "^4.2.0",
     "uuidv4": "^6.2.13",
-    "ws": "^8.8.0"
+    "ws": "^8.8.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.18.6",
-    "@babel/preset-env": "^7.18.6",
-    "@mojaloop/api-snippets": "^14.0.0",
+    "@babel/core": "^7.18.9",
+    "@babel/preset-env": "^7.18.9",
+    "@mojaloop/api-snippets": "^14.2.0",
```

Comment:
I believe it does now since the Interface is now part of the API Snippet --> https://github.com/mojaloop/sdk-scheme-adapter/pull/340/files#diff-8edf0cb22838239441d348ba38b95b757052d8834f21c1ee61b07a9a77174106R93

Reply:
Along with this, I have deprecated the [package/@mojaloop/sdk-scheme-adapter](https://www.npmjs.com/package/@mojaloop/sdk-scheme-adapter), with the following message provided by @kleyow.

```
The use case of this library was to support Typescript types for the sdk-scheme-adapter's outbound interface for integration purposes. This functionality is now superseded by @mojaloop/api-snippets which now hosts the outbound interface. api-snippets generates and exports these types and can be imported using "import { SDKSchemeAdapter } from '@mojaloop/api-snippets'"
```

Reply:
Yes, it's a requirement now:
https://github.com/mojaloop/sdk-scheme-adapter/blob/85207fc5c265f73cc3e33075e3019c5561130b5f/src/OutboundServer/index.js#L93

Reply:
I do have a concern here about moving the Interfaces from the SDK into the api-snippets, specifically around the “discoverability” of the Interface for anybody who is interested in using the Outbound API.

@kirgene please make sure you update the README to clearly include links to both Inbound and Outbound interfaces for consistency.

Reply:
@mdebarros have updated README